tests/sentry/api/test_event_search.py

```diff
@@ -1412,5 +1429,6 @@ def test_orderby_field_aggregate(self):
             ["uniq", "user", "count_unique_user"],
             ["argMax", ["id", "timestamp"], "latest_event"],
             ["argMax", ["project.id", "timestamp"], "projectid"],
+            ["transform(projectid, [], [], '')", None, "project.name"],
```

Comment:
Should we have a test that includes projects, or are there integration tests covering that?

Reply:
Integration tests are sorta covering this, but you're right. Probably better to have a test on the event_search as well.